providers/jaxb/src/main/java/org/jboss/resteasy/plugins/providers/jaxb/AbstractJAXBContextFinder.java

```diff
@@ -148,9 +173,14 @@ public JAXBContext createContext(Annotation[] parameterAnnotations, Class... cla
 				if (type == null)
 					continue;
 				classes1.add(type);
-				Class<?> factory = findDefaultObjectFactoryClass(type);
-				if (factory != null)
-					classes1.add(factory);
+               try {
```

Comment:
Method, findDefaultObjectFactoryClass can be throwing the exception so this is correct.